### PR TITLE
make install: install containerd-shim-runc-v2 + remove useless constants

### PR DESCRIPTION
**Pull request by AkihiroSuda**
**- What I did**
Added `containerd-shim-runc-v2` (used by cgroup2 mode) to `make install`

**- How to verify it**
`make install`

**- A picture of a cute animal (not mandatory but encouraged)**
🐧

### CODE REVIEW

File: hack/make/.binary-setup [hunk 1]
```diff
@@ -5,6 +5,7 @@ DOCKER_RUNC_BINARY_NAME='runc'
 DOCKER_CONTAINERD_BINARY_NAME='containerd'
 DOCKER_CONTAINERD_CTR_BINARY_NAME='ctr'
 DOCKER_CONTAINERD_SHIM_BINARY_NAME='containerd-shim'
+DOCKER_CONTAINERD_SHIM_RUNC_V2_BINARY_NAME='containerd-shim-runc-v2'
```
Comment (thaJeztah) [inline hack/make/.binary-setup:8]: We should consider getting rid of these env-vars; I don't think they're really used

**Review by thaJeztah — APPROVED** [review]
LGTM

File: hack/make/.binary-setup (hunk 1 above)
Comment (AkihiroSuda) [reply hack/make/.binary-setup]: done

**Review by thaJeztah — APPROVED** [review]
LGTM

ping @cpuguy83 (in case you were using these variables 😅)

**Review by cpuguy83 — APPROVED** [review]
LGTM

**Pull request merged**